tests/lib/tools/suite/tests.backup/task.yaml

```diff
@@ -0,0 +1,52 @@
+summary: tests for the tests.backup tool
+
+prepare: |
+    mv "${PWD}.tar" "${PWD}.tar.back"
```

Comment:
`PWD` is a full path so this would be `/home/zyga.tar`.

Reply:
sorry, don't get what you want to change here

Reply:
This is just a comment, no need to take action.